Documentation/contributing.rst

```diff
@@ -1715,26 +1724,39 @@ If you intent to release a new minor release, see the
             <<end of add-to-NEWS.rst>>
 
 #. Add all modified files using ``git add`` and create a pull request with the
-   title ``Prepare for release v1.0.3``. Add the label ``stable/backport`` to
-   the PR.
+   title ``Prepare for release v1.0.3``. Add the backport label to the PR which
+   corresponds to the branch for which the release is being performed, e.g.
+   ``backport/1.0``.
 
    .. note::
 
        Make sure to create the PR against the desired stable branch. In this
        case ``v1.0``
 
-#. Follow standard procedures to get the
+
+#. Follow standard procedures to get the aforementioned PR merged into the 
```

Comment:
Should we link to http://docs.cilium.io/en/v1.1/contributing/#pull-request-review-process here?